docs/updating/updating_to_3.3.md

```diff
@@ -0,0 +1,138 @@
+# Updating [[= product_name =]] to v3.3
+
+[[= product_name =]] v3.3 uses [Symfony Flex](https://symfony.com/doc/current/quick_tour/flex_recipes.html).
+When updating from v3.2 to v3.3, you need to follow a special update procedure.
+
+If you are updating from an earlier version, start with updating your installation to v3.2.
+
+First, create an upgrade branch in git and commit your work.
+
+!!! tip
+
+    It is good practice to make git commits after every step of the update procedure.
+
+## Merge composer.json
+
+Merge the special update branch into your project:
+
+```
+git merge origin/v3.2-to-v3.3-upgrade
+```
+
+This will introduce changes from the [website skeleton](https://github.com/ibexa/website-skeleton/blob/main/composer.json)
+and result in conflicts in `composer.json`.
+
+Resolve the conflicts in the following way:
+
+- Make sure all `ezsystems/*` packages are removed.
```

Comment:
I've talked with @webhdx and it's not that easy.

User should remove most of the of `ezsystems/*` packages, but if there are any packages created by us that are not part of a standard installation (for example: translations packages, e-content package) that were added later explicitly by the user then they need to be kept.

Reply:
`Make sure all automatically added ezsystems/* packages are removed. If you explicitly added any packages that are not part of the standard installation, retain them.` < sound OK?

Reply:
Sounds good to me!